inputfiles/addedTypes.jsonc

```diff
@@ -409,7 +409,7 @@
                         "createObjectURL": {
                             "exposed": "Window Worker",
                             "overrideSignatures": [
-                                "createObjectURL(object: any): string"
+                                "createObjectURL(object: File | Blob | MediaSource): string"
```

Comment:
This is actually not needed anymore, please remove the whole URL item here (from line 405 to line 417) and then rebuild.

Reply:
Roger. I removed URL, re-ran `npm run build && npm run baseline-accept` and sent out a new commit.

Appreciate your friendly feedback!